.travis/install_isitgr_linux.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+git clone https://github.com/mishakb/ISiTGR
+cd ISiTGR/camb
+python setup.py install
```

Comment:
Don't do this.  Just do `pip install isitgr` here.
Or fix the setup.py script in the isitgr repo.  It's currently broken.  (Doesn't install the .so file along with the .py files.)  The pip installation works though, so just do that here.

Reply:
Ok. Thanks for getting back to me. Before you respond, we got isitgr into conda-forge so I tried it and it seem to find isitgr now just fine, but there seems to be some troubles finding the data  files like model0_pk_MG_matterpower.dat. I am working on another deadline today so I will check this again tomorrow morning.